unittests/test_pipeline.py

```diff
@@ -573,7 +573,24 @@ def test_unknown_system(self):
                 'value3': (3.1, -0.1, 0.1),
             }
         }
-        self.assertRaises(SanityError, self.test.check_performance)
+        self.test.check_performance()
+
+    def test_unknown_multiple_systems(self):
```

Comment:
I don't think this unit test offers sth more compared to the `test_unknown_system` one.

In one of the reference numbers you should add a unit as well and we are still missing sth to test that the units are emitted correctly.

Reply:
The idea behind this unit test was checking the ability of the code to collect different variables from different systems and include all of them in the added default case.

Reply:
I guess we could merge then this and the previous one. Just add another unknown system to the previous one.